### DISCUSSION TITLE
Unblock成功了，但是步骤很繁琐，不知道是不是我的操作有问题？

### DISCUSSION THREAD

**Discussion by YunfangHou** [1715173329/UnblockNeteaseMusic#621] [Q&A]
Unblock成功了，但是步骤很繁琐，不知道是不是我的操作有问题？
我现在 Unblock 的一套完整流程是，打开 Terminal，运行命令
```bash
open -a Proxifier;
cd UnblockNeteaseMusic;
yarn;
yarn build;
env QQ_COOKIE="uin=xxxx; qm_keyst=xxxxxxxx" node app.js -o qq
```
然后再打开网易云音乐，才可以。而且这个 Terminal 窗口还不能关闭。这样的话，我为了听音乐，要同时挂着三个软件（网易云音乐、Proxifier、Terminal）。
不知道是不是我的操作方式有问题？有更简单的使用方式吗？

**Comment by 1715173329** [comment]
安装为服务，用 iptables 劫持网易云音乐流量即可。

**Reply by YunfangHou** [reply]
我是macOS，没有可运行文件吧
